.npmrc

```diff
@@ -1,2 +1,3 @@
 auto-install-peers=true
 save-exact=true
+@fluencelabs:registry=https://npm.fluence.dev/
```

Comment:
why is this necessary? shouldn't it work with the default npm registry?

Reply:
We are waiting for the fix from machines team. Once it's get done we will switch to release version

Reply:
Fixed now!